ReactCommon/jsi/jsi/jsi.h

```diff
@@ -252,6 +252,7 @@ class JSI_EXPORT Runtime {
 
   virtual String createStringFromAscii(const char* str, size_t length) = 0;
   virtual String createStringFromUtf8(const uint8_t* utf8, size_t length) = 0;
+  virtual Object createArrayBufferFromBytes(const void* bytes, size_t length) = 0;
```

Comment:
Nit: this declaration probably shouldn't be in the middle of the string-related methods.

Reply:
I moved it a bit further below. Does that work?